<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbIndentation">
     <property name="title">
      <string>Indentation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="rbIndentationTabs">
          <property name="text">
           <string>Tabs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbIndentationSpaces">
          <property name="text">
           <string>Spaces</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="lIndentationWidth">
          <property name="text">
           <string>Width :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sIndentationWidth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>16</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lnIndentationWidth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="numDigits">
           <number>2</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAutodetectIndent">
     <property name="text">
      <string>Try to detect indentation automatically</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>544</width>
       <height>214</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sIndentationWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lnIndentationWidth</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
